From 40137c006c341542fec8be139b3acea78afb584e Mon Sep 17 00:00:00 2001
From: Eric Koegel <eric.koegel@gmail.com>
Date: Sat, 6 Dec 2014 20:16:32 +0300
Subject: [PATCH 09/12] Revert "Fix crash on logout/login via SunRay"

This reverts commit 417e387750920b9e092705863a863e71a77ea431.

Causes a regression with polkit authorizations. See:
https://github.com/ConsoleKit2/ConsoleKit2/issues/16
https://github.com/ConsoleKit2/ConsoleKit2/issues/12
---
 src/ck-seat.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/ck-seat.c b/src/ck-seat.c
index 7f79fea..6e6800f 100644
--- a/src/ck-seat.c
+++ b/src/ck-seat.c
@@ -514,9 +514,7 @@ change_active_session (CkSeat    *seat,
          * must be sent when the database dump is finished it is
          * important that the '-full' signalled is emitted first. */
 
-        if (CK_IS_SESSION (old_session)) {
-                g_signal_emit (seat, signals [ACTIVE_SESSION_CHANGED_FULL], 0, old_session, session);
-        }
+        g_signal_emit (seat, signals [ACTIVE_SESSION_CHANGED_FULL], 0, old_session, session);
         g_signal_emit (seat, signals [ACTIVE_SESSION_CHANGED], 0, ssid);
 
         if (old_session != NULL) {
-- 
2.2.1

